src/app/loaders/express/index.ts

```diff
@@ -169,6 +170,8 @@ const loadExpressApp = async (connection: Connection) => {
   // Registered routes with the Singpass/Corppass servers
   app.use('/singpass/login', SingpassLoginRouter)
   app.use('/corppass/login', CorppassLoginRouter)
+  // jwks endpoint for SP OIDC
+  app.use('/singpass/.well-known/jwks.json', SpOidcJwksRouter)
```

Comment:
nit: i feel like this should have been in the /api/v3/singpass router but i guess its too late :(